frontend/src/App.js

```diff
@@ -102,8 +96,9 @@ class Posts extends Component {
     super(props);
   }
   Item() {
```

Comment:
Move item to own component. e.g. `Post` and instead of `for` looping through the posts use `.map` function to loop through the items.

e.g. in `Posts`:
```js
<Row>
  {this.props.items.map(post => <Post key={post.id} {...post} />)}
</Row>
```